ledger/ledger-utxo/src/main/java/net/corda/v5/ledger/utxo/transaction/UtxoTransactionBuilder.java

```diff
@@ -185,16 +185,16 @@ public interface UtxoTransactionBuilder {
      * @return Returns the notary assigned to the current transaction, or null if the notary has not been set.
      */
     @Nullable
-    Party getNotary();
+    MemberX500Name getNotary();
```

Comment:
```suggestion
    MemberX500Name getNotaryName();
```

Consistency with other API classes in this PR. Also removes ambiguity.